renderer/components/dashboard/__tests__/RunTestCard.test.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import React from 'react'
+import { screen, render, cleanup, fireEvent } from '@testing-library/react'
+import { theme } from 'ooni-components'
+import { ThemeProvider } from 'styled-components'
+import { IntlProvider } from 'react-intl'
+import English from '../../../../lang/en.json'
+import { useRouter } from 'next/router'
+
+import RunTestCard from '../RunTestCard'
+import { testList } from '../../nettests'
+
+// Mocking useRouter()
+const mockPush = jest.fn()
+jest.mock('next/router', () => ({
+  useRouter() {
+    return {
+      route: '/',
+      pathname: '/dashboard',
+      push: mockPush
+    }
+  },
+}))
+
+const renderComponent = (component, locale = 'en', messages = English) => {
+  return render(
+    <IntlProvider locale={locale} messages={messages}>
+      <ThemeProvider theme={theme}>{component}</ThemeProvider>
+    </IntlProvider>
+  )
+}
+
+describe('Tests for RunTestCard component', () => {
+  const router = useRouter()
+
+  afterEach(() => {
+    cleanup()
+    jest.clearAllMocks()
+  })
+
+  test('All the Test Cards are mounted', async () => {
```

Comment:
This test is relevant only if we are testing by rendering `pages/dashboard/index.js` and checking if all there are cards for each item in `testList`. Otherwise we are manually mounting all the cards in the test setup itself.